<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BenchmarkDialog</class>
 <widget class="QDialog" name="BenchmarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基准点位置</string>
  </property>
  <widget class="QComboBox" name="comboBox_Block">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>15</y>
     <width>76</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>23</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>块选择</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>78</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>1.位置类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>2.标记位置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_move">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>190</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_get">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>260</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_OutLight">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>470</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>440</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>7.光线</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_ok">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>20</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>470</y>
     <width>38</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>外光：</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_cancel">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>100</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>260</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>4.形状</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_InLight">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>510</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>510</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>内光：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>23</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>拼板选择</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Array">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>15</y>
     <width>81</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_positioningType">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>70</y>
     <width>76</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_MarkPos">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>461</width>
     <height>131</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>90</number>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>26</number>
   </attribute>
   <column>
    <property name="text">
     <string>No</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>X</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Shape</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Result</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>260</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>5.搜索面积</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>400</y>
     <width>46</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>6.分数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Score">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>390</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>460</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>500</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>临界值：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CriticalValue">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>100</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>46</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3.参数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Binary">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>540</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>二进位</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_auto">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>自动</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>350</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>尺寸Y：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>300</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>尺寸X：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ShapeX">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>290</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>450</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>极性：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>400</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ARM：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ShapeY">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>340</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Arm">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>390</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>500</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>厚度：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Polarity">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>440</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>白色</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Thickness">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>490</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>550</y>
     <width>42</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>旋转：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Rotate">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>540</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>300</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AreaX">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>290</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AreaWidth">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>290</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>宽度X：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AreaY">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>340</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>350</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>宽度Y：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AreaHeight">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>340</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Outline">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>490</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>轮廓线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Scan">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>340</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>扫描</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AutoAdjust">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>440</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>自我调整</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Test">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>390</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Position">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>100</width>
     <height>36</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>位置：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>550</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>8.轨道</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cantilever">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>540</y>
     <width>100</width>
     <height>36</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>78</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>2.形状</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Shape">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>461</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
